functions/weather.location.fish

```diff
@@ -6,36 +6,34 @@ function weather.location -d "Get the current geographic location"
   end
 
   # Fetch location data based on our IP
-  if not set geoip_data (weather.fetch "http://geoip.nekudo.com/api/$ip")
+  if not set geoip_data (curl -s "https://ipapi.com/ip_api.php?ip=$ip")
     echo "Unable to query GeoIP data; please try again later."
     return 1
   end
-
+  
```

Comment:
Please remove this from the diff, it's not part of the fix.

Reply:
This structures the code, visually.